cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
+title: Estimate Serverless Usage with MovR
+summary: Run the MovR sample application to estimate RU usage in CockroachDB Serverless.
+toc: true
+referral_id: docs_serverless_movr
+docs_area: get_started
+---
+
+This tutorial shows you how to initialize and run the MovR workload on a {{ site.data.products.serverless }} cluster, then use the Request Unit consumption data to estimate a sufficient budget for the MovR workload.
+
+## Overview
+
+MovR is a fictional company that offers users a platform for sharing vehicles, like scooters, bicycles, and skateboards, in select cities across the United States and Europe. The MovR team wants to use {{ site.data.products.serverless }} to run their high-growth, variable workload. They decide to test out a sample workload to see how it will perform and how much it will cost to run.
```

Comment:
suggest linking first use of 'movr' in this paragraph to https://www.cockroachlabs.com/docs/v22.1/movr.html